modules/oxide/src/less/theme/components/form/textfield.less

```diff
@@ -56,7 +56,8 @@
   .tox-textfield:focus {
     background-color: @textfield-focus-background-color;
     border-color: @textfield-focus-border-color;
-    box-shadow: @textfield-focus-box-shadow;
+    border-radius: 6px;
```

Comment:
Don't we have variables for this. Isn't there skins that have sharp borders? See if there is some variable that this could be using.

Reply:
Reverted